src/Microsoft.Restier.EntityFramework/DbApi.cs

```diff
@@ -60,14 +60,13 @@ protected T DbContext
         [CLSCompliant(false)]
         protected override IServiceCollection ConfigureApi(IServiceCollection services)
         {
-            services = base.ConfigureApi(services)
+            return base.ConfigureApi(services)
                 .CutoffPrevious<IModelBuilder>(ModelProducer.Instance)
                 .CutoffPrevious<IModelMapper>(new ModelMapper(typeof(T)))
                 .CutoffPrevious<IQueryExpressionSourcer, QueryExpressionSourcer>()
-                .CutoffPrevious<IQueryExecutor>(QueryExecutor.Instance)
+                .ChainPrevious<IQueryExecutor, QueryExecutor>()
```

Comment:
Yeah we'd have to make DefaultQueryExecutor the innermost, since we removed the executor replacement in DefaultQueryHandler, now provider IQueryExecutor could receive an EnumerableQuery, which can't be handled by provider, and so provider should delegate it to inner executor (DefaultQueryExecutor).